### ISSUE TITLE
Hubs Multi Servier Template Update fails / No SES emails - Software Version 1.1.5 

### ISSUE THREAD

**Issue by standtech** [mozilla/hubs#6002]
**Description**

**To Reproduce**
1. Go to '[CloudFormation]'
2. Click on 'Update.'
3. Scroll down to ' No of app servers' or 'No of Steams servers'
4. change number 
5. See error 

In the events You see - 
AppASG | UPDATE_IN_PROGRESS | New instance(s) added to autoscaling group - Waiting on 1 resource signal(s) with a timeout of PT30M.
-- | -- | --

StreamASG | UPDATE_IN_PROGRESS | New instance(s) added to autoscaling group - Waiting on 1 resource signal(s) with a timeout of PT30M.
-- | -- | --

And it fails / rolls back  in the end . 
That means we can not scale no of servers during the actual event . Same behaviour when I changed Admin email as well . So it looks like , anything which is an update will fail the deployment. 

Another issue is ,  First time deployment works but Mails are not received which are required for admin login even though N.virginia has domain/email verified in SES  ( in sandbox)  so not able to login even after 9-10 attempts .
PS: Personal Edition worked without any hiccups . 

**Expected behavior**

As per the documentation , it should scale . vertical or horizontal EC2 scaling. Allows administrator to tweak number and Ec2 types based on use case , Development scenario or Production events. 

After Deployment, you browse admin console and key in Email ID configured. A link is send to you admin email and you click and validate authenticity to login on Admin console. 

**Screenshots**

<img width="955" alt="Screen Shot 2023-03-20 at 4 32 00 pm" src="https://user-images.githubusercontent.com/128365495/226300113-90a24293-58b5-4b1e-8a88-40f17222bba8.png">

**Hardware** 
 - Device: MacBook 
 - OS: MacBook has Monterey , on AWS , Default AWS AMIs bundled in template 
 - Browser: [Chrome/Safari ) 

**Additional context**

**Comment by SV-AndreiC** [comment]
I was able to successfully update a stack, adding extra instances while using a Windows machine. I was also running the vanilla Hubs Cloud on the stack.
![image](https://user-images.githubusercontent.com/93765706/226641273-ae96541a-6153-4aa7-ac7f-0ee6faa00983.png)

Are u using a custom client or the vanilla version?

**Comment by standtech** [comment]
I am running Enterprise multi-server on AWS cloud using template . After many permutations and combinations,  it seems it works with Root account but not with an account with full admin role assigned.  So its either AWS Policy or cloudformation internal calls issue.  Not much documentation around this or clear instructions.

**Comment by eyesnareinc** [comment]
I also encountered stalling (I think) with AppASG with the same Status reason message as above: "New instance(s) added to autoscaling group - Waiting on 1 resource signal(s) with a timeout of PT30M" The update resulted in a stack update fail. Update just was moving the stack offline to online. I tried to update multiple times and the stack failed to update and triggered UPDATE_ROLLBACK_COMPLETE each time. Then, I updated the template to Hubs Cloud Personal 1.1.5 (stack was offline) and the stack update failed at AppDb with the status reason "Cannot find upgrade target from 10.serverless_21 with requested version 11.13 for serverless engine mode." Rollback also failed.

**Comment by eyesnareinc** [comment]
![hubscloudrollback1](https://user-images.githubusercontent.com/66790476/226735193-d12b6d13-1747-47d9-9ead-d8ef9700d83a.JPG)

**Comment by standtech** [comment]
I am doing Fresh installation and  facing again same issue with Root account as well , no respite.
It's likes If it go through you are lucky , just keep trying .  ;)

**Comment by SV-AndreiC** [comment]
@tanfarming Can u take a look, QA team was not able to reproduce this issue with the lastest HC version and the 1.1.5 template.

**Comment by standtech** [comment]
PS: We are using Amazon Linux AMIs through the Mozilla hubs software.

**Comment by standtech** [comment]
Tried few times more  , both multi and single instance fresh install , am still getting "502 Bad Gateway" due to below issue .

<img width="1207" alt="Screen Shot 2023-03-23 at 1 36 10 am" src="https://user-images.githubusercontent.com/128365495/227004672-81a9183a-d4de-407a-8067-e48d3fab7b61.png">

**Comment by standtech** [comment]
Installed Personal edition , after fresh install got black screen saying cloud is down , and when tried to update to turn off failed with the same issue. 

![image](https://user-images.githubusercontent.com/128365495/227134297-584a2ee8-cdc0-4798-9db9-3eb433d0401c.png)

**Comment by eyesnareinc** [comment]
Same error happened to me on 2 brand new stack builds.
![forBrandonP2](https://user-images.githubusercontent.com/66790476/227320127-ed7c14dd-4ac7-4e5b-89dc-363ab56af059.JPG)

**Comment by standtech** [comment]
Hello 

When can we get some response here , seems product is not stable and impossible to deploy.

**Comment by standtech** [comment]
HELLO MOZILLA HUBS SUPPORT

ITS JUNE NOW , Could we have any response on this ? 

Product is not a open source so expecting some resolution here...
